records/generator.go

```diff
@@ -448,6 +484,9 @@ func slaveIdTail(slaveID string) string {
 // zk://username:password@host1:port1,host2:port2,.../path
 // file:///path/to/file (where file contains one of the above)
 func getProto(pair string) (string, string, error) {
```

Comment:
I find it confusing with call this function getProto when we're using it with ip:port pairs too. We're essentially just splitting and returning the tokens as different return arguments so this is generalizable.

```go
func split(s, sep string) (string, string, error) {
    ps := strings.SplitN(s, sep, 2)
    if len(ps) != 2 {
        return "", "", fmt.Errorf("%q not found in %q", sep, s)
    }
    return ps[0], ps[1], nil
}
```